cves/CVE-2016-2161.yml

```diff
@@ -66,15 +62,14 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
    - commit: db5fcaac2a76d88329dbb014de0664b5b0a30bfb
-     note:
+     note: 
```

Comment:
What about this commit fixed it?

Reply:
There was a pretty comprehensive comment in the commit that explained it, but you're right I should add it here as well - done.